experiments/urls.py

```diff
@@ -6,8 +6,10 @@
 from experiments.views import MeasurementXLSImportView, XLSTemplateDownloadView, WholeFileXLSImportView, \
     UUIDGeneratorView, UUIDAndCreateModeInjectorView
 
+from django.views.generic import TemplateView
+
 urlpatterns = [
-    path('', RedirectView.as_view(url='/import/'), name='go-to-import'),
+    path('', TemplateView.as_view(template_name="home.html"), name='home'),
```

Comment:
RedirectView import is not needed anymore, it must be removed. I see unused imports in Pycharm, I think you can install a plugin for that in VSCode too

Reply:
`urls.py` header is not needed too